internal/diag.go

```diff
@@ -92,6 +92,9 @@ func Run(params Params) error {
 		"clusterroles.txt": func(writer io.Writer) error {
 			return kubectl.Describe("clusterroles", "elastic", "", writer)
 		},
+		"clusterrolebindings.txt": func(writer io.Writer) error {
+			return kubectl.Describe("clusterrolebindings", "elastic", "", writer)
```

Comment:
Do we want to filter by prefix or label? @pebrc

Reply:
I think prefix is fine for now.